app/middleware.go

```diff
@@ -97,6 +105,7 @@ func (l LoggerMiddleware) Serve(next echo.HandlerFunc) echo.HandlerFunc {
 			zap.String("ip", ip),
 			zap.String("method", method),
 			zap.String("path", path),
+			zap.String("caller", fmt.Sprintf("%v", caller)),
```

Comment:
Since we are not identifying the caller, but the game ID, you can use "gameID" to match what @thiagomachado did [here](https://git.topfreegames.com/game-tools-and-services/chat-vendor-interface/-/merge_requests/211/diffs#b07e17ac84c0b0934178ac91b62e8efa2b74d96b_35_39). Does it make sense?